src/platform/silabs/tracing/BUILD.gn

```diff
@@ -0,0 +1,16 @@
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+static_library("SilabsTracing") {
```

Comment:
why the static_library vs a source_set?

Reply:
In case we want to have the tracing available on public repos someday since this code is SL-ONLY